src/Extensions/Filters/PropertyNameFilter.php

```diff
@@ -103,6 +103,11 @@ public function getPropertyName($property, $lang = null)
      */
     public function getPropertySelectionValueName($property, $lang = null)
     {
+        $propertyValueDecoded = json_decode($property['value'], true);
+        if(is_array($propertyValueDecoded) && count($propertyValueDecoded)) {
+            $property['value'] = array_slice($propertyValueDecoded, 0, 1)[0];
```

Comment:
Würde hier nicht `array_shift` ausreichen? Dürfte zumindest schneller sein, als erst einen Array-Ausschnitt zu erstellen, um die erste Position zu ziehen.
```suggestion
            $property['value'] = array_shift($propertyValueDecoded);
```